### PR TITLE
No fakes

### PR DESCRIPTION
**Pull request by nishp1**
https://app.asana.com/0/45511869812564/64233498710418

Depends on https://github.com/percolate/eslint-config-percolate/tree/jsx_indent

### CODE REVIEW

**Comment by maxkostow** [comment]
@nishp1 this shouldn't be based on the #12 as that branch will require a lot of updates to hotlanta.

**Comment by nishp1** [comment]
I also needed to use the updated API in v1.10 of eslint. So, I branched of
off the work you already did.

On Mon, Dec 7, 2015 at 6:07 PM, Max Kostow <notifications@github.com> wrote:

> @nishp1 <https://github.com/nishp1> this shouldn't be based on the #12
> <https://github.com/percolate/eslint-config-percolate/pull/12> as that
> branch will require a lot of updates to hotlanta.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/percolate/eslint-config-percolate/pull/13#issuecomment-162700354>
> .
>



-- 
Nish Patel
Frontend Engineer
Get inspired. See what's new on the Percolate blog: blog.percolate.com

**Comment by maxkostow** [comment]
the lets update the versions without the jsx stuff

File: test/example.spec.js [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+    var sandbox
+
+    beforeEach(function () {
+        sandbox = sinon.sandbox.create()
+        sandbox.spy()
+        sandbox.stub()
+        sandbox.mock()
+    })
+    afterEach(function () {
+        sandbox.resotre()
```
Comment (maxkostow) [inline test/example.spec.js]: Is this typo on purpose?

Comment (nishp1) [inline test/example.spec.js]: Nope, not on purpose.

**Comment by nishp1** [comment]
Ok, created another PR https://github.com/percolate/eslint-config-percolate/pull/14

**Pull request closed without merging**